.travis.yml

```
@@ -20,3 +20,4 @@ addons:
       - libgif-dev
       - librsvg2-dev
       - g++-4.9
+cache: npm
```

missing node_modules
```suggestion
cache: npm
  bundler: true
  directories:
  - node_modules
```